app/models/concerns/patient_reportable.rb

```diff
@@ -35,8 +43,9 @@ module PatientReportable
       end
     end
 
-    def self.ltfu_time_ago(date)
-      (date.to_date - LTFU_TIME).end_of_month
+    # Returns a plain timestamp to be interpreted in reporting time, for use with materialized views
+    def self.reporting_timestamp(time)
```

Comment:
Add this to `Period`?

Reply:
We're not sure where this will go yet, Period isn't the right place because it doesn't really deal with timezones or "reporting". There are a few other things like decided which timezone, using that timezone, etc, in the reporting path that will need extraction as well, but it's not clear where they'll go yet.

We'll keep this here for now, and keep an eye out for when/where to extract these. This is a workaround, and is not meant to set the precedent for other places/contexts.